### DISCUSSION TITLE
Connecting to Janeway Preprint Platform

### DISCUSSION THREAD

**Discussion by BruceCaron** [knowledgefutures/pubpub#1280] [Ideas]
Let me know if you want to work to connect the preprint review feature with the new Janeway Preprint Platform.  EarthArXiv is using this, hosted by the CDL.

**Comment by gabestein** [comment]
Hi Bruce! Yeah, we'd love to chat about it! Please reach out to hello@pubpub.org and we'll put together a call.

**Reply by BruceCaron** [reply]
I've reached out to the CDL Janeway folks.... stay tuned.

**Reply by gabestein** [reply]
Awesome, thanks!

**Reply by BruceCaron** [reply]
Do you need to access the Janeway API for this backend connection?  Any info on that from your end?

**Reply by gabestein** [reply]
We're trying to take an ecosystem approach, rather than building one-off integrations. The way we've made it work with various arxiv's so far is we deposit reviews to the crossref api with `isReviewOf` relationships pointing to the preprint. Then, we provide a feed of the community's deposits from the crossref API, which can be filtered by the arxiv to check for relations to their preprints. We've found this works pretty well because most arxiv's already have integrations with crossref to build on. And in this way, you can support any platform that can deposit those relationships rather than just PubPub.

**Reply by BruceCaron** [reply]
Does this work with journals too? or a sub-set of these?

**Reply by gabestein** [reply]
It depends how you mean...but it should work with any content that's deposited to crossref with any [relationship type](https://www.crossref.org/education/content-registration/structural-metadata/relationships/). And relationships can exist (more or less) between any two content types. For example, we're depositing some reviews against preprints on SSRN that were deposited as journal articles.
